src/views/Gallery.vue

```diff
@@ -68,35 +68,38 @@ export default defineComponent({
    setup() {
     const store = useStore();
     return {
-      loading: computed(() => store.getters["feed/loading"]),
-      latestPosts: computed(() => store.getters["feed/entries"]),
+      loading: computed(() => store.getters["gallery/loading"]),
+      pictures: computed(() => store.getters["gallery/entries"]),
       store,
       loadMore: (ev: CustomEvent) => {
-        store.dispatch("feed/loadMore").then(() => {
+        store.dispatch("gallery/loadMore").then(() => {
           (ev.target as any).complete();
         });
       },
+      refresh() {
+        store.dispatch("gallery/refresh");
+      },
     }
   },
   computed:{
-    filteredPosts(): any[] {
-      const postsWithPictures = this.latestPosts.filter((post: any) =>
-        post.objects.some((object: any) => object.className === "Picture")
-      );
-
-        if (this.searchValue.length!==0 && this.searchValue!=='All' && this.searchValue!=='setting') {
-          return postsWithPictures.filter((post: any) =>
-            post.team.name.toLowerCase().includes(this.searchValue.toLowerCase())
-          );
-        }
-        else{
-            return postsWithPictures
-        }
-    },
     filteredPictures(): any[] {
-      return this.filteredPosts.flatMap((post: any) => post.objects)
+      if (
+        this.searchValue.length!==0 &&
+        this.searchValue!=='All' &&
+        this.searchValue!=='setting'
+      ) {
+        return this.pictures.filter((post: any) =>
+          post.team.name.toLowerCase().includes(this.searchValue.toLowerCase())
+        );
+      }
+      else{
+          return this.pictures
+      }
```

Comment:
I was wondering what this is about, then I realised dropping the search-field wasn't clearly communicated. So, lemme do that: search is currently not making any sense - just remove it. Then you can drop this and just link the team-selector to the `store.dispatch('gallery/selectTeam',...` (or so) action and don't have to care about messing with that here.... this is store stuff...